ui/lib/core/addon/components/search-select.js

```diff
@@ -126,6 +131,14 @@ export default Component.extend({
       this.handleChange();
     },
     selectOption(option) {
+      if (this.selectLimit) {
+        if (
+          this.selectLimit === this.selectedOptions.length ||
```

Comment:
This can be simplified to `if (this.selectLimit <= this.selectedOptions.length) ...`

Reply:
You're a 🧙‍♂️ when it comes to simplification of conditionals.  Thanks!